packages/analytics/analytics-geo-map/vite.config.ts

```diff
@@ -18,11 +18,11 @@ const config = mergeConfig(sharedViteConfig, defineConfig({
     },
     rollupOptions: {
       // Make sure to externalize deps that shouldn't be bundled into your library
-      external: ['maplibre-gl', '@kong-ui-public/i18n'],
+      external: ['maplibre-gl'],
       output: {
         // Provide global variables to use in the UMD build for externalized deps
         globals: {
-          '@kong-ui-public/i18n': 'kong-ui-public-i18n',
+          'maplibre-gl': 'maplibre-gl',
```

Comment:
is this how maplibre-gl exposes their global in the browser?

Reply:
The syntax here is:

```
'actual package name': 'package name variable as exposed on globalThis'
```

So as an example, we expose `@kong/icons` as `KongIcons` in globals (you can see the declaration [here](https://github.com/Kong/icons/blob/dc5fc0bab7eddd5301ca9367e0565b9a479e4592/vite.config.ts#L45)).

So you would need to look up how `maplibre-gl` exposes its global variable in the browser, which I believe is located [here](https://github.com/maplibre/maplibre-gl-js/blob/ba966429637d7ffa2dd7296b8a99cea0bc38e686/rollup.config.csp.ts#L17)

So the correct entry in our repo would look like this
```suggestion
          'maplibre-gl': 'maplibregl',
```